### DISCUSSION TITLE
I think there was a mistake

### DISCUSSION THREAD

**Discussion by junnylc** [wqweto/VbAsyncSocket#22] [Q&A] [answered]
First, thank you for your program.
In the file ctxWinSock.ctl, Private Sub m_oSocket_OnSend(), “RaiseEvent SendProgress(m_lSendPos, UBound(m_baSendBuffer) + 1)” Should be modified to “RaiseEvent SendProgress(m_lSendPos, UBound(m_baSendBuffer) + 1 - m_lSendPos)”. What do you think?

**Comment by wqweto** [comment, marked as answer]
10x! Fixed in 3747b4ea5656d89c0c769b8c664c1ae3936312fa

**Reply by junnylc** [reply]
really?maybe i download an old version

**Reply by wqweto** [reply]
@junnylc I fixed this after you reported it.

You can send Pull Requests with your fixes too if you want to contribute to the project.
